From 4d24f9b5ab7fddad2c62a6ce47ce9542835a3981 Mon Sep 17 00:00:00 2001
From: Sebastian Bazley <sebb@apache.org>
Date: Thu, 5 Nov 2015 00:36:29 +0000
Subject: [PATCH] VALIDATOR-374 Email Validator : .school domain is being
 rejected Was already fixed - add unit test to show this

git-svn-id: https://svn.apache.org/repos/asf/commons/proper/validator/trunk@1712698 13f79535-47bb-0310-9956-ffa450edef68
---
 src/changes/changes.xml                                       | 4 ++++
 .../apache/commons/validator/routines/EmailValidatorTest.java | 4 ++++
 2 files changed, 8 insertions(+)

diff --git a/src/changes/changes.xml b/src/changes/changes.xml
index 9024859ad..a48ff048a 100644
--- a/src/changes/changes.xml
+++ b/src/changes/changes.xml
@@ -43,6 +43,10 @@ The <action> type attribute can be add,update,fix,remove.
   <body>
 
   <release version="1.5.0" date="tba" description="tba">
+    <action issue="VALIDATOR-374" type="fix">
+    Email Validator : .school domain is being rejected
+    Add Unit test to show it has been fixed
+    </action>
     <action issue="VALIDATOR-376" type="fix" dev="rgoers">
       Revert EmailValidator to handle top level domains to the behavior prior to VALIDATOR-273. Allow an optional
       behavior to allow the behavior VALIDATOR-273 implemented. Note that this is a behavioral change for users
diff --git a/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java b/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java
index e6140a622..8297ad9a3 100644
--- a/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java
+++ b/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java
@@ -500,4 +500,8 @@ public void testValidator365() {
                 "Etiamnisinislvenenatisvelauctorutullamcorperinjusto.Proinvelligulaerat.Phasellusvestibulumgravidamassanonfeugiat."+
                 "Maecenaspharetraeuismodmetusegetefficitur.Suspendisseamet@gmail.com"));
     }
+
+    public void testValidator374() {
+        assertTrue(validator.isValid("abc@school.school"));
+    }
 }
